app/views/generic_object_definition/show_list.html.haml

```diff
@@ -1,2 +1,8 @@
 #main_div
   = render :partial => 'layouts/gtl'
+
+  %generic-object-definition-toolbar#generic_object_definition_show_list_form
+
+:javascript
+  miq_bootstrap('#generic_object_definition_show_list_form');
```

Comment:
@himdel : we discussed this as an anti-pattern. Do we have a replacement yet?

Reply:
Why would `miq_bootstrap` on the angular component be an anti-pattern?
@himdel Pray tell me how.

Reply:
This is an anti-pattern simply because a GTL show-list should not have any other components .. what is this `generic-object-definition-toolbar` actually for? Should it not be an actual toolbar? 

Also, it's a component without a template - sounds like something is very wrong there :).

Reply:
Aah.. and it is an invisible toolbar because you're actuall trying to do things right and call the API for all the toolbar actions, am I right? :)

Sounds maybe we could just move this outside the DOM, since it has no effect on any actual html...

And we can actually listen for `Rx` messages even outside Angular code.. so maybe this should be a simple file that just starts listening for messages wherever - it means we would have the listeners on every screen but... so what :).

We just need the messages to be discoverable - but `genericObjectDefinitionToolbarController` pretty much solves that.

Reply:
I mean.. instead of using a component (and having a non-standard `show_list`), this could just be a js file which just starts the listeners directly, without anything angulary.

Reply:
> GTL show-list should not have any other components

Seems very restrictive for no real reason.
Why wouldn't be more components allowed? 

> generic-object-definition-toolbar

` generic-object-definition-toolbar` is the controller for the toolbar commands. 
BTW, there are no groundbreaking changes here.
Something similar has already been implemented for Physical Servers - https://github.com/ManageIQ/manageiq-ui-classic/pull/1380

> Also, it's a component without a template - sounds like something is very wrong there 

I have one simple answer to that - Delete operation does not need a form/template. A flash message is good enough
Nothing went wrong ;)

Reply:
> I mean.. instead of using a component (and having a non-standard show_list), this could just be a js file which just starts the listeners directly, without anything angulary.

(Looks like I missed this message since I did not refresh the page as I was typing mine)

- It cannot be non-angulary because I'm using the `API` service, which is available only if you use angular.
- Also there is some advantage to using the the controller/component approach that I see.
All messages that are triggered by the Generic Object Definition toolbar would only be directed to the observable in the `generic-object-definition-toolbar` component.
It's like I'm channeling the messages in a particular area of interest and not going in the common application-wide listeners pool.

Reply:
@AparnaKarve :  You are correct, there's no other way to do this right now.  It is 100% ok for this PR. 

The comment was ment more as a note that this is something we have to deal with and ping to @himdel to look for a solution ;-)

I am looking at all the "extra" content in the "show" partials because I would like to delete all the "show" partials on our way towards more declarative and client-side UI.

Reply:
@martinpovolny Thanks for that clarification.